### PR TITLE
Fix undeclared references to FS classes

### PR DESCRIPTION
**Pull request by tom95**

### CODE REVIEW

**Comment by tom95** [comment]
This is a fix to the issue opened for Squot https://github.com/hpi-swa/Squot/issues/276

File: MonticelloTonel-FileSystem.package/.squot-contents [hunk 1]
```diff
@@ -1,4 +1,4 @@
 SquotTrackedObjectMetadata {
-	#serializer : #SquotCypressCodeSerializer,
-	#objectClassName : #PackageInfo
+	#objectClassName : #PackageInfo,
+	#serializer : #SquotCypressCodeSerializer
```
Comment (j4yk) [inline MonticelloTonel-FileSystem.package/.squot-contents:3]: Looks like I should investigate why this order changes... But maybe it is only to recent changes in Squeak's Dictionary.

**Review by j4yk — APPROVED** [review]

**Comment by j4yk** [comment]
Looks like Travis is not active for this repository yet. Anyway I think these changes are safe enough.

**Comment by tom95** [comment]
I believe we're at looking at a couple of changes from STON that will trickle into all our repos at some point. Another change that I have noticed pop up occasionally concerned how double quotes are escaped. I believe this was an upstream change in STON, but I haven't tracked it down precisely yet, maybe this one? https://github.com/squeak-smalltalk/squeak-ston/commit/ef44950bf09137600c483783db43731dfdcc5238

**Comment by fniephaus** [comment]
I just set up a GitHub workflow for this repo in order to test the PR before the merge. @j4yk beat me to it...

**Pull request merged**